frontend/src/scenes/onboarding/OnboardingStep.tsx

```diff
@@ -13,21 +14,38 @@ export const OnboardingStep = ({
     children: React.ReactNode
 }): JSX.Element => {
     const { onboardingStep, totalOnboardingSteps } = useValues(onboardingLogic)
-    const { incrementOnboardingStep, completeOnboarding } = useActions(onboardingLogic)
+    const { incrementOnboardingStep, decrementOnboardingStep, completeOnboarding } = useActions(onboardingLogic)
     return (
-        <BridgePage view="onboarding-step" noLogo hedgehog={false} fixedWidth={false}>
+        <BridgePage
+            view="onboarding-step"
+            noLogo
+            hedgehog={false}
+            fixedWidth={false}
+            header={
+                onboardingStep > 1 && (
+                    <div className="mb-4">
+                        <LemonButton icon={<IconArrowLeft />} onClick={decrementOnboardingStep}>
+                            Back
+                        </LemonButton>
+                    </div>
+                )
+            }
+        >
             <div className="max-w-md">
-                <h1>{title}</h1>
+                <h1 className="font-bold">{title}</h1>
                 <p>{subtitle}</p>
                 {children}
-                <LemonButton
-                    type="primary"
-                    onClick={() =>
-                        onboardingStep == totalOnboardingSteps ? completeOnboarding() : incrementOnboardingStep()
-                    }
-                >
-                    {onboardingStep == totalOnboardingSteps ? 'Finish' : 'Continue'}
-                </LemonButton>
+                <div className="mt-8 flex justify-end gap-x-2">
+                    <LemonButton
+                        type="primary"
+                        onClick={() =>
+                            onboardingStep == totalOnboardingSteps ? completeOnboarding() : incrementOnboardingStep()
+                        }
+                        sideIcon={onboardingStep !== totalOnboardingSteps ? <IconArrowRight /> : null}
+                    >
+                        {onboardingStep == totalOnboardingSteps ? 'Finish' : 'Continue'}
```

Comment:
Separate to this PR but I wondered if I should be brought back to the "Choose a product" page rather than being redirected to the Session Replay product once I'd finished. Maybe it's worth encouraging people to set up more products but feel free to ignore if that comes at a different point in a customers onboarding e.g. when they first click on a product

Reply:
In an upcoming PR there will be a screen that allows people to enable other products right away. Here's a very, very rough idea:

![image](https://github.com/PostHog/posthog/assets/18598166/ce5e6f4c-96d5-42db-a073-118cb7528348)

But I do like the idea of giving people an option to set up another product. I will add that, probably in another PR since that's where I'll be working on the "additional products" section.